keyboards/geistmaschine/macropod/readme.md

```diff
@@ -0,0 +1,25 @@
+# Ada
+
+![geist](imgur.com image replace me!)
```

Comment:
Missing the image

Reply:
Thanks, forgot that... Added pic and updated readme in general.